parallelization_test/parallelization-benchmark.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<p>How about creating a quick function out of this code chunk as it's also repeated below?</p><pre class="ql-syntax" spellcheck="false">def calculate_time(n_jobs):
  []
  return times_many
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>